src/components/SearchMultiSelectInput/index.tsx

```diff
@@ -38,6 +40,8 @@ export type Props<
     sortFunction?: (options: O[], search: string, labelSelector: (option: O) => string) => O[];
     onSearchValueChange?: (value: string) => void;
     onShowDropdownChange?: (value: boolean) => void;
+    selectedOptionContainerClassName?: string;
+    showSelectedList?: boolean;
```

Comment:
Let's change the prop name to something like `listShown`
@AdityaKhatri @frozenhelium any thoughts?

Reply:
`selectionListShown`